salt/_modules/metalk8s.py

```diff
@@ -615,3 +622,36 @@ def get_from_map(value, saltenv=None):
         input_data=tmplstr,
         saltenv=saltenv
     )
+
+
+def configure_archive(archive, removed=False):
+    """Add (or remove) a MetalK8s archive in the bootstrap config file."""
+    # raise if the archive does not exist or is invalid
+    archive_info_from_product_txt(archive)
+
+    try:
+        with open(BOOTSTRAP_CONFIG, 'r') as fd:
+            config = yaml.safe_load(fd)
+    except IOError as exc:
+        msg = 'Failed to load bootstrap config file at "{}": {}'.format(
+            BOOTSTRAP_CONFIG, str(exc)
+        )
+        raise CommandExecutionError(message=msg)
```

Comment:
Not link to this PR but in Python3 it's better to raise exception and use `from exc` to have both exceptions (check this https://www.python.org/dev/peps/pep-3134/#explicit-exception-chaining)

NOTE: I fix all other except in my pylint PR #3066 (e.g.: https://github.com/scality/metalk8s/pull/3066/commits/e4557955353e02b360fe71926dbf1760d5ce5cc1)

Reply:
Oh, didn't know that, I will change it.
Thanks for the tip.